### DISCUSSION TITLE
Content traversing API in a template file (👍21)

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#3980] [Enhancement Queue]
Surfaced via @fvsch on Twitter: https://twitter.com/fvsch/status/1043918915406057472

![image](https://user-images.githubusercontent.com/39355/45951316-a953b580-bfc8-11e8-841d-ed11afcde269.png)

Inspired by https://getkirby.com/

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

The enhancement backlog can be found here: https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+

Don’t forget to upvote the top comment with 👍!

**Comment by ollicle** [comment]
👍 for this, _especially_ if the content folders include those _created_ by the build.

I hope to use this to create index files in date based directories e.g. when creating `/bog/2020/12/30/directory-indexes-in-11ty/index.html` I could generate  indexes such as`/blog/2020/index.html` and `/blog/2020/12/index.html`.

Would be necessary to also find a means to associate a templates to be used with these generated indexes.

**Comment by edwardhorsford** [comment]
@ollicle You can do date based archives with a custom collection - [I included some details here](https://github.com/11ty/eleventy/issues/316).

**Comment by ollicle** [comment]
Thanks again, @edwardhorsford 
I don’t know if the documentation has changed since I last attempted to tackle this (https://github.com/11ty/eleventy/issues/502), but it is finally clicking for me how pagination may be used to generate indexes. 🚀
